### PR TITLE
Add an explicit -repositoryRoot (-R) option.

### PR DESCRIPTION
**Pull request by peichman-umd**
Fixes https://jira.duraspace.org/browse/FCREPO-2741

### CODE REVIEW

File: src/main/java/org/fcrepo/importexport/ArgParser.java [hunk 1]
```diff
@@ -193,6 +193,13 @@
                 .desc("Path to the bag config file")
                 .build());
 
+        configOptions.addOption(Option.builder("R")
+                .longOpt("repositoryRoot").argName("uri")
```
Comment (bseeger) [inline src/main/java/org/fcrepo/importexport/ArgParser.java]: argName("uri") is called twice here.

Comment (peichman-umd) [reply src/main/java/org/fcrepo/importexport/ArgParser.java]: I was following the model of the other options, where `argName` is called for both the long opt and short option.

Comment (bseeger) [reply src/main/java/org/fcrepo/importexport/ArgParser.java]: Odd, it doesn't seem like calling it twice would do anything unique here. _But_ I can't say I fully understand what Option.Builder is doing in the first place.

File: src/main/java/org/fcrepo/importexport/ArgParser.java [hunk 2]
```diff
@@ -193,6 +193,13 @@
...
+                .hasArg(true).numberOfArgs(1).argName("uri")
+                .required(false)
+                .desc("Repository root URI")
```
Comment (dbernstein) [inline src/main/java/org/fcrepo/importexport/ArgParser.java]: @peichman-umd :  Can you expand the description to include the intent of this feature, including whether or not it applies to both import and export modes as well as the default behavior when it is not specified?

Comment (peichman-umd) [reply src/main/java/org/fcrepo/importexport/ArgParser.java]: @dbernstein Opened new PR at #116

**Pull request closed without merging**